src/com/dotmarketing/business/web/WebContext.java

```diff
@@ -0,0 +1,48 @@
+package com.dotmarketing.business.web;
```

Comment:
The "com/dotmarketing/business" package is where legacy code lives. This new class must be placed inside the new "com/dotcms/api" packages.